adr/0017-hardware-screening-os.md

```diff
@@ -0,0 +1,34 @@
+# 0017. Hardware screening for OS
+
+Date: 2020-10-23
+
+## Status
+
+Draft
+
+## Context
+
+We need take a decision for the minimal requirements of a IoT-Board to become a Home Assistant OS image.
+Also how we deal with old or replaced generation/boards.
+
+## Decision
+
+Follow are the requirements of a Board:
+
+- The board need to be available for everbody
+- The manifacture produce and sell the board at least 3 years
+- The manifacture is know for his continuity
+- The board need to be supported by mainstream Linux kernel with current or next LT
+- The board need supported by u-boot/barebox bootloader
```

Comment:
Is no upstream U-Boot/Barebox support required?

Reply:
it should, yes